format-definitions/parsec-corim-tpm.md

```diff
@@ -0,0 +1,119 @@
+# Parsec TPM endorsement
+
+This document covers the technical details of the metadata produced by the Parsec service for endorsing the platform to Veraison.
+
+The endorsement will allow the Veraison-based verifier to trust and verify attestation tokens produced by this platform. Two endorsements are needed: one to identify the identity key used by the platform to sign attestation tokens, and another one to specify the reference values expected for a class of platforms. The identity key endorsement must be provided for each platform that needs to attest itself, while the platform endorsement is only needed once per class of platforms. A class of platforms is a group of devices expected to run identical software stacks, which would thus produce identical quotes as platform attestation tokens.
+
+## TPM key endorsement format
+
+Endorsements for a Veraison-based verifier are provided in a CoRIM format. In our use-case, the endorsements for platform identity keys are JSON encoded using the template below.
+
+```json
+{
+    "tag-identity": {
+            "id": uuid // example: uuid = "00000000-0000-0000-0000-000000000000"
+        },
+    "entities": [
+        {
+            "name": "PARSEC",
+            "regid": "https://github.com/parallaxsecond",
+            "roles": [
+                "tagCreator",
+                "creator",
+                "maintainer"
+            ],
+        }
+    ],
+    "triples": {
+        "attester-verification-keys": [
+            {
+                "environment": {
+                    "class": {
+                        "type": "uuid",
+                        "value": class // example: class = "ffffffff-ffff-ffff-ffff-ffffffffffff"
+                    },
+                    "instance": {
+                        "type": "ueid",
+                        "value": key_id, // example: key_id = "f05eae32-0003-4a82-8cfa-3780ea7817a0"
```

Comment:
the example is not correct (it's a [`ueid` of type random](https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat#table-1) not a UUID